pool/app/mailing/entity.ml

```diff
@@ -12,6 +12,12 @@ module StartAt = struct
   let schema = schema field create
 end
 
+module StartNow = struct
```

Comment:
Refactor to:

```ocaml
module Start = struct
  type t =
    | StartNow
    | StartAt of Ptime.t

  let to_ptime = function
    | StartAt start_at ->
      if Ptime.is_earlier ~than:(Ptime_clock.now ()) start_at
      then Error Pool_common.Message.TimeInPast
      else Ok start_at
    | StartNow -> Ok (Ptime_clock.now ())
  ;;

  let of_string = ...
end
```

Reply:
Please have a look at this commit: https://github.com/uzh/pool/pull/115/commits/6eb8751468f363df56249e58cff620c3328e710f

I am not sure if this solution is any better 😄